tensorflow/workspace.bzl

```diff
@@ -537,6 +537,7 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
 
   # TODO(phawkins): currently, this rule uses an unofficial LLVM mirror.
   # Switch to an official source of snapshots if/when possible.
+  # FIXME: The LLVM Mirror should now have polly checked-out as well.
```

Comment:
Not clear what this FIXME is getting at -- is it that the LLVM mirror has polly checked out and we should figure out some way to avoid that?  Or the converse?

Reply:
No, I meant to say that once the Polly build file integration is complete, the Polly source code should be checked-out in the LLVM mirror used for TensorFlow.